components/brave_wallet_ui/nft/nft.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
```

Comment:
Under which `Site`/domain this will be executed? Is the image going to be served in the same site?

Reply:
@thypon This will be executed on `chrome-unstrusted://nft-display`. The page expects a query param `imageUrl` which is used as `src` attribute for the image in the page.